subcommand/injector/flags.go

```diff
@@ -160,6 +167,11 @@ func (c *Command) init() {
 	c.flagSet.StringVar(&c.flagResourceLimitMem, "memory-limit", agent.DefaultResourceLimitMem,
 		fmt.Sprintf("Memory resource limit set in injected containers. Defaults to %s", agent.DefaultResourceLimitMem))
 
+	c.flagSet.StringVar(&c.flagTLSMinVersion, "tls-min-version", defaultTLSMinVersion,
+		fmt.Sprintf(`Minimum supported version of TLS. Defaults to %s. Accepted values are "tls10", "tls11", "tls12" or "tls13"`, defaultTLSMinVersion))
```

Comment:
I wonder if we should take these values from `tlsutil.TLSLookup` rather than hard coding them here? When tls14 is supported then that would be a valid as in input but not documented in the help.

Reply:
Yeah, I like that idea, added in 892ba54. It comes out looking like this:

```
  -tls-min-version=<string>
     Minimum supported version of TLS. Defaults to tls12. Accepted
     values are tls10, tls11, tls12, tls13.
```